### DISCUSSION TITLE
Umbraco - Webhook - Overview - Checkbox is pretty confusing

### DISCUSSION THREAD

**Discussion by JeffreyPerplex** [umbraco/Umbraco-CMS#15234] [Features and ideas]
In the logoverview of Umbraco 13 webhooks I think it's pretty confusing that there's a checkbox. I would expect that I could check multiple checkboxes and then perform a bulk action. But as soon as you click on the checkbox it goes to the detailpage of the log issue:

![afbeelding](https://github.com/umbraco/Umbraco-CMS/assets/12457148/eb95b4f8-981f-44df-a6ab-c004644da443)

I would suggest changing it to "True/False"

**Comment by bjarnef** [comment]
I agree this can lead to confusing, but I think the `<umb-checkmark>` component need to be redesigned a bit as it is too close to `<umb-checkbox>` component. Note this is also used for default language in languages section and in user or user groups picker IIRC.

@iOvergaard @nielslyngsoe I think the design need to be re-considered.
